src/Rector/Class_/PreferPHPUnitSelfCallRector.php

```diff
@@ -95,6 +99,19 @@ public function refactor(Node $node): ?Node
                 return null;
             }
 
+            $callerType = $this->nodeTypeResolver->getType($node->var);
+            if (! $callerType instanceof TypeWithClassName) {
+                return null;
+            }
+
+            $classReflection = $this->reflectionResolver->resolveClassReflectionSourceObject($node);
+            if ($classReflection instanceof ClassReflection && $classReflection->hasNativeMethod($methodName)) {
+                $method = $classReflection->getNativeMethod($methodName);
+                if (! $method->isStatic()) {
```

Comment:
use positive check, then return nodefactory seems ok here, else, return null

Reply:
Is now included in latest. version.